src/Makefile.test.include

```diff
@@ -60,6 +60,7 @@ BITCOIN_TESTS =\
   test/merkle_tests.cpp \
   test/miner_tests.cpp \
   test/multisig_tests.cpp \
+  test/net_tests.cpp \
```

Comment:
No strong opinion: IMO we don't need to use an identical filename for what is tested in the test. I think `addrman_tests.cpp` would be more clear.